pkg/controller/node/nodecontroller_test.go

```diff
@@ -326,6 +399,9 @@ func TestMonitorNodeStatusEvictPods(t *testing.T) {
 			evictionTimeout, util.NewFakeRateLimiter(), util.NewFakeRateLimiter(), testNodeMonitorGracePeriod,
 			testNodeStartupGracePeriod, testNodeMonitorPeriod, nil, false)
 		nodeController.now = func() unversioned.Time { return fakeNow }
+		for _, ds := range item.daemonSets {
```

Comment:
I started out doing it that way, but the caches never get populated during the test because we never call the method that fires up those goroutines.  If you look down to TestCheckPod, you'll see we're doing the same thing there with nodes, and it's presumably for the same reason.